Updating billing page language (#4413)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/billing.md

In addition, you can look at the Job Details page's Insights tab to show how many models are being built per month for that particular job and which models are taking the longest to build. 

> Usage information is available to customers on consumption-based plans, and not all usage visualizations may be visible to customers on legacy plans. Any usage data shown in dbt CLoud is only an estimate of your usage, and there may be a delay in showing usage data in the product &mdash; your final usage for the month will be visible on your monthly statements (statements applicable to Team and Enterprise plans).

Reviewer: i was taught that:

- "may" can mean a few things so is not translation friendly. avoid using it when might/could/should can be used instead. something like "you may contact support if you have questions" is ok tho. 
- em dashes should be used for clauses but not for complete sentences

```suggestion
Usage information is available to customers on consumption-based plans, and some usage visualizations might not be visible to customers on legacy plans. Any usage data shown in dbt Cloud is only an estimate of your usage, and there could be a delay in showing usage data in the product. Your final usage for the month will be visible on your monthly statements (statements applicable to Team and Enterprise plans).
```
